cohort_banking_initiator_js/src/banking-app.ts

```diff
@@ -25,13 +25,14 @@ export class BankingApp {
 
     async init() {
         this.initiator = await Initiator.init(sdkConfig)
```

Comment:
If kafka is down the error will bubble into main method. There is an example how we can handle typed errors.